packages/runtime/test-runtime-utils/src/mocks.ts

```diff
@@ -518,16 +534,26 @@ export class MockFluidDataStoreRuntime
 		return null;
 	}
 
+	private submitMessageInternal(messageContent: any, localOpMetadata: unknown): number {
+		return this.containerRuntime?.submit(messageContent, localOpMetadata) ?? 0;
```

Comment:
It's a bit hard for me to follow logic RE presence of containerRuntime. Do we drop op on the floor if it's not present? Should this class always have access to container runtime instance (maybe empty implementation)?

Reply:
Unless we mix and match versions within the mocks, this should never happen. I can also do an assert to make this more explicit. This is so that we can actually merge these changes into main. In next, these should not be optional. The issue is that the mocks are part of the public API.